src/utils/domUtils.ts

```diff
@@ -20,13 +20,13 @@ export module Utils {
       return bbox;
     }
 
-    export var POLYFILL_TIMEOUT_MILLISECONDS = 1000 / 60; // 60 fps
+    export var REQUEST_ANIMATION_FRAME_TIMEOUT_MILLISECONDS = 1000 / 60; // 60 fps
```

Comment:
We were arguing this should not be exported? I could not find a way not to export it, unless actually using the `requestAnimationFramePolyfill` itself